test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Kindly check your `.gitignore` file, because your `test.md` file is not being ignored properly. If your `.gitignore` file works properly, you should not be able to see any `test.md` in your GitHub branch or PR.